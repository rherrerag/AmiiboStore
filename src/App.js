import React from 'react';
import {NativeBaseProvider} from 'native-base';
import {NavigationContainer} from '@react-navigation/native';
import AppStackNavigator from './navigation';
import {Provider} from 'react-redux';
import store from './redux/store';

const App = () => {
  return (
    <Provider store={store}>
      <NativeBaseProvider>
        <NavigationContainer>
          <AppStackNavigator />
        </NavigationContainer>
      </NativeBaseProvider>
    </Provider>
  );
};

export default App;
