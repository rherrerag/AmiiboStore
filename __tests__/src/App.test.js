import React from 'react';
import {render} from '@testing-library/react-native';
import App from '../../src/App';

describe('App container tests', () => {
  test('deberia renderizar componente App correctamente', () => {
    const {toJSON} = render(<App />);
    expect(toJSON()).toMatchSnapshot();
  });
});
