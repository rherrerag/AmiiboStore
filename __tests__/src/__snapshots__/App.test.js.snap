// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App container tests deberia renderizar componente App correctamente 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }
/>
`;
